api/src/interpreter/interpreter.service.ts

```diff
@@ -44,6 +44,12 @@ export class InterpreterService {
 
     query = paginateInterpreterQueryDto.filter(query);
 
+    if (language) {
+      query
+        .leftJoinAndSelect('interpreter.languages', 'int_languages')
+        .leftJoinAndSelect('int_languages.language', 'languages_lang')
+    }
+
```

Comment:
let me note here: This is for get all langs from relation. not just filtered results.